src/ui/templates/results/alternativeverticals.hbs

```diff
@@ -1,28 +1,14 @@
 <div class="yxt-AlternativeVerticals{{#unless isShowingResults}} yxt-AlternativeVerticals--notShowingResults{{/unless}}">
   <div class="yxt-AlternativeVerticals-noResultsInfo">
-    <em class="yxt-AlternativeVerticals-noResultsInfo--emphasized">
-      No results found
-    </em>
-    in {{currentVerticalLabel}}.
+    {{translate phrase='<em class="yxt-AlternativeVerticals-noResultsInfo--emphasized">No results found</em>in [[currentVerticalLabel]].' currentVerticalLabel=currentVerticalLabel }}
```

Comment:
Oh does this just work as is? I was thinking we might have to update the runtimeTranslation helper to return a SafeString or something like that but haven't tested anything

Reply:
This sort of mixed markdown works in the jambo translator. @alexisgrow Did your manual testing include this translation?

Reply:
@oshi97 OMG lol I was too focused checking the interpolation, I was like, yeah this looks right to me 😮 
<img width="863" alt="Screen Shot 2020-09-02 at 1 30 04 PM" src="https://user-images.githubusercontent.com/39104038/92016521-7079fa80-ed20-11ea-8036-4ec84cd4c143.png">

Updated so that HTML is no longer escaped; screenshot in commit message is how things currently look!